src/components/Map/Map.js

```diff
@@ -1,24 +1,46 @@
-import { useState, useEffect } from "react";
-import { createMap, setCurrentLocationAsCenter } from "services/MapService";
+import { useState, useEffect } from 'react';
+import { GoogleMap, useLoadScript } from '@react-google-maps/api';
 import "components/Map/Map.scss";
```

Comment:
```suggestion
import 'components/Map/Map.scss';
```
you should be able to configure this with eslint

Reply:
I'll fix them and I'll add the configuration in [this PR](https://github.com/RobertoCarrilloAvila/target/pull/19)